doc/source/api/data.rst

```diff
@@ -13,6 +13,7 @@ Inference library converters
 
    from_cmdstan
    from_cmdstanpy
+   from_beanmachine
```

Comment:
It should be the first, we try to sort elements within the same API heading to be alphabetically sorted